CHANGELOG.md

```diff
@@ -11,6 +50,7 @@ Very minor patch release to fix the full size AWS tests and re-run the template
 - Remove traces of markdownlint in the template ([#1486](https://github.com/nf-core/tools/pull/1486)
 - Remove accidentally added line in `CHANGELOG.md` in the template ([#1487](https://github.com/nf-core/tools/pull/1487))
 - Update linting to check that `.editorconfig` is there and `.yamllint.yml` isn't.
+- Don't save md5sum for `versions.yml` when running `nf-core modules create-test-yml` ([#1511](https://github.com/nf-core/tools/pull/1511))
```

Comment:
Is correct that this change is under the previous tools version?

Reply:
Yup, well spotted both - fixed https://github.com/nf-core/tools/pull/1583